components/team-codes/MyTeams.js

```diff
@@ -1,27 +1,31 @@
+import React, { useState } from "react";
+import {
+  Image,
+  Text,
+  View,
+  StyleSheet,
+  TouchableOpacity,
+  TextInput
+} from "react-native";
 
-import React, { useState } from 'react';
-import { Image, Text, View, StyleSheet, TouchableOpacity, TextInput } from 'react-native';
+import { getTeamsForUserId } from "../../utils/db/teams";
+import KitText from "../KitText";
 
-import { getTeamsForUserId } from '../../utils/db/handle'
-import KitText from '../KitText';
-
-const myUserId = 'user_id_1'; //EVAN TODO. This is hardcoded for now, just to see if linkup to firebase will work. We can use var userId = firebase.auth().currentUser.uid; later
+const myUserId = "user_id_1"; //EVAN TODO. This is hardcoded for now, just to see if linkup to firebase will work. We can use var userId = firebase.auth().currentUser.uid; later
 
 export function MyTeams(props) {
-  const [myTeams, setMyTeam] = useState([])
-  getTeamsForUserId(myUserId)
+  const [myTeams, setMyTeam] = useState([]);
+  getTeamsForUserId(myUserId);
 
   return (
-    <View style={{marginTop: 10,}}>
-      {myTeams.length == 0 && <KitText 
-                                style={{}} 
-                                size={36}>
-                                  No Teams Yet
-                              </KitText>}
+    <View style={{ marginTop: 10 }}>
+      {myTeams.length == 0 && (
```

Comment:
I'll keep this styling in mind why doing these conditional displays

Reply:
Oh yeah it's just because I have autoformat on save and I saved this file. You should install the Prettier extension on VS Code btw.